app/models/similar_occupation_standards.rb

```diff
@@ -21,18 +21,44 @@ def query
     {
       size: RESULTS_SIZE,
       query: {
-        more_like_this: {
-          fields: ["title"],
-          like: [
+        bool: {
+          should: [
+            {match: {
+              title: {query: occupation_standard.title, boost: 5}
+            }},
+            {match: {
+              work_process_titles: {
+                query: occupation_standard.work_processes.pluck(:title).to_sentence,
```

Comment:
This is probably not the best way to query on this array, but is passing tests now, I will look into how to refactor it